Formula/slides.rb

```diff
@@ -0,0 +1,36 @@
+class Slides < Formula
+  desc "Terminal based presentation tool"
+  homepage "https://github.com/maaslalani/slides"
+  url "https://github.com/maaslalani/slides/archive/refs/tags/v0.1.2.tar.gz"
+  sha256 "195f5269ef78733081fa0034add5dca9c5c855ee01bd990795741cb70e385f23"
+  license "MIT"
+  head "https://github.com/maaslalani/slides.git", branch: "main"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args
+  end
+
+  test do
+    markdown = <<-SLIDES
```

Comment:
Can we write this to a file instead, and open the file with `slides`? Pipes sometimes produce orphan processes that `brew` isn't able to kill after the test times out, and this makes some CI runs deadlock.

Reply:
👍

Reply:
Done 🙂